soh/soh/z_message_OTR.cpp

```diff
@@ -187,4 +187,142 @@ extern "C" void OTRMessage_Init()
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_REWARD_PIECE_OF_HEART,
+        CustomMessage("Astounding! A whopping %y5000%w&rupees saved! For such a feat, a&%rPiece of Heart%w is yours!",
+                    "Erstaunlich! Satte %y5000%w Rupien&gespart! Für eine solche Leistung&gehört Ihnen ein %rStück Herz%w!",
+                    "Étonnant! Un énorme %y5000%w roupies&économisées! Pour un tel exploit,&un %rFragment de Coeur%w!"));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_ZERO_AMOUNT,
+        CustomMessage(
+            "Eh, @? You can't&deposit the air in your pockets!&Come back when you've got some&real Rupees.",
+            "Äh, @? Sie können die&Luft nicht in Ihren Taschen&deponieren! Kommen Sie zurück,&wenn Sie echte Rupien haben.",
+            "Hein, @? Vous ne&pouvez pas déposer l'air dans vos&poches ! Revenez quand vous&aurez de vrais rubis."
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_INSUFFICIENT_BALANCE,
+        CustomMessage(
+            "Oh, @! It seems your&balance is running low. You need&more Rupees in the bank to make&this withdrawal.",
+            "Oh, @! Es scheint, dass&Ihr Guthaben zur Neige geht. Für&diese Auszahlung benötigen Sie&mehr Rupien auf der Bank.",
+            "Oh, @! Il semble que&votre solde soit faible. Vous avez&besoin de plus de roupies en&banque pour effectuer ce retrait."
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_WALLET_FULL,
+        CustomMessage(
+            "Whoa there, @! That&amount's too hefty for your wallet.&Trim it down a bit, eh?",
+            "Wow, @! Dieser Betrag&ist zu hoch für Ihren Geldbeutel.&Reduzieren Sie es ein wenig, oder?",
+            "Waouh, @! Ce montant&est trop lourd pour votre&portefeuille. Réduisez-le un peu,&hein?"
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_MAX_BALANCE,
+        CustomMessage(
+            "Woah, @! The bank's&max capacity is %y5000%w Rupees. I&don't have enough room for that&amount!",
+            "Boah, @! Die maximale&Kapazität der Bank beträgt %y5000%w&Rupien. Ich habe nicht genug Platz&für diese Menge!",
+            "Waouh, @! La capacité&maximale de la banque est de&%y5000%w roupies. Je n'ai pas assez de&place pour ce montant!"
+        ));
+    CustomMessageManager::Instance->CreateMessage(
+        customMessageTableID, TEXT_BANKER_ERROR_DEPOSIT_NOT_WORTHWHILE,
+        CustomMessage(
+            "Hold up, @! There's a %y{{rupeesFee}}%w&Rupee handling fee. You'd lose&Rupees, not gain! Deposit more to&make it worthwhile.",
+            "Warte, @! Es fällt eine&Bearbeitungsgebühr von %y{{rupeesFee}}%w Rupien&an. Du würdest Rubine verlieren!&Zahlen Sie bitte mehr ein.",
```

Comment:
Considering the majoritarian usage of the polite form in the added dialogue lines, I would suggest to standardize the German dialogue in a consistent way.

`Du würdest Rubine verlieren!` &#8594; `Sie würden Rubine verlieren!`